proto/src/determined/api/v1/config_policies.proto

```diff
@@ -0,0 +1,97 @@
+syntax = "proto3";
+
+package determined.api.v1;
+option go_package = "github.com/determined-ai/determined/proto/pkg/apiv1";
+
+import "google/protobuf/struct.proto";
+
+// PutWorkspaceConfigPoliciesRequest is a complete replacement of config
+// policies for the workspace and workload type.
+message PutWorkspaceConfigPoliciesRequest {
+  // Specifies which workspace the config policies apply to. Use global API for
+  // global config policies.
+  int32 workspace_id = 1;
+  // Specifies which task type the config policies apply to: EXPERIMENT or NTSC.
+  string workload_type = 2;
+  // The config policies to use. Contains both invariant configs and constraints
+  // in yaml or json format.
+  string config_policies = 3;
+}
+
+// Response to PutWorkspaceConfigPolicies request.
+message PutWorkspaceConfigPoliciesResponse {
+  // The config policies saved. Contains both invariant configs and constraints
+  // in yaml or json format.
+  google.protobuf.Struct config_policies = 1;
+}
+
+// PutGlobalConfigPoliciesRequest is a complete replacement of global config
+// policies for the workload type.
+message PutGlobalConfigPoliciesRequest {
+  // Specifies which task type the config policies apply to: EXPERIMENT or NTSC.
+  string workload_type = 1;
+  // The config policies to use. Contains both invariant configs and constraints
+  // in yaml or json format.
+  string config_policies = 2;
+}
+
+// Response to PutGlobalConfigPolicies request.
+message PutGlobalConfigPoliciesResponse {
+  // The config policies saved. Contains both invariant configs and constraints
+  // in yaml or json format.
+  google.protobuf.Struct config_policies = 1;
+}
+
+// GetWorkspaceConfigPoliciesRequest is used to list all task config policies
+// for the workspace and task type.
```

Comment:
```suggestion
// GetWorkspaceConfigPoliciesRequest lists the task config policies
// (the invariant config and constraints)  of type workload_type set on a given workspace.
```

^ I sort of like specifying the invariant config and constraint rather than saying "all" task policies since for a given workspace and workload type, there can't be more than 2!